app/models/spree/slide.rb

```diff
@@ -31,4 +38,8 @@ def slide_link
   def slide_image
     !image.file? && product.present? && product.images.any? ? product.images.first.attachment : image
   end
+
+  def in_time?
+    Time.now.between?(starts_at || 1.second.ago, ends_at || 1.second.from_now)
```

Comment:
Favor the use of `Time.current` instrad of `Time.now`

![image](https://user-images.githubusercontent.com/237597/30175038-c306801c-93c2-11e7-8204-0a7727728dc3.png)
[Source](https://www.varvet.com/blog/working-with-time-zones-in-ruby-on-rails/)